src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/crafting/SlimefunCraftingListener.java

```diff
@@ -32,7 +27,7 @@ default boolean isUnallowed(@Nullable ItemStack item) {
         }
 
         SlimefunItem sfItem = SlimefunItem.getByItem(item);
-        return !(sfItem instanceof VanillaItem) && !sfItem.isDisabled();
+        return sfItem != null && !(sfItem instanceof VanillaItem) && !sfItem.isDisabled();
```

Comment:
This is the exact same as line 34.
So you can very much utilise this method:

```suggestion
        return isUnallowed(sfItem);
```